wolfcrypt/src/ecc.c

```diff
@@ -5938,6 +5960,11 @@ int wc_ecc_init_label(ecc_key* key, const char* label, void* heap, int devId)
             ret = BUFFER_E;
     }
 
+#if defined(HAVE_PKCS11)
+    XMEMSET(key, 0, sizeof(ecc_key));
```

Comment:
Same , must be done after `wc_ecc_init_ex`

Reply:
Kept it the same for reasons discussed on slack